server/db/009-addbatteryleveltosessions.sql

```diff
@@ -0,0 +1,20 @@
+DO $migration$
```

Comment:
You're going to end up in conflict with @risatronic 's DB migration. You'll need to end up changing this to `migrationId := 10` and the file name change accordingly.